src/rabbit_cli.erl

```diff
@@ -125,9 +135,22 @@ main(ParseFun, DoFun, UsageMod) ->
             rabbit_misc:quit(?EX_SOFTWARE)
     end.
 
+start_distribution_anon(0, LastError) ->
+    {error, LastError};
+start_distribution_anon(TriesLeft, _) ->
+    NameCandidate = list_to_atom(rabbit_misc:format("rabbitmq-cli-~2..0b", [rabbit_misc:random(100)])),
```

Comment:
Yeah, you'll need https://github.com/rabbitmq/rabbitmq-common/pull/48
And also you can't simply run ./scripts/rabbitmqctl from rabbitmq-server checkout - rabbit_common will be missing from code path (but on installed server it will be available through rabbit_common.ez archive).